a3p-integration/proposals/f:test-orchestration/initial.test.js

```diff
@@ -3,21 +3,21 @@ import test from 'ava';
 import { agd, getVatDetails } from '@agoric/synthetic-chain';
 
 const vats = {
-  network: { incarnation: 0 },
-  ibc: { incarnation: 0 },
-  localchain: { incarnation: 0 },
+  network: { incarnation: 1 },
+  ibc: { incarnation: 1 },
+  localchain: { incarnation: 1 },
   orchestration: { incarnation: 0 },
-  transfer: { incarnation: 0 },
+  transfer: { incarnation: 1 },
```

Comment:
these bumps are caused by `e:upgrade-next`, right? we should test it in that proposal.

the other tests of `f:orchestration` are things that should always work so they should move into `z:acceptance`.

I think we shouldn't have a separate `test-orchestration` "proposal".

Reply:
Sounds good, I'll make that happen.